src/components/Article.jsx

```diff
@@ -1 +1,47 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const firstArticle = {
+  title: 'Electric-propulsion startup Orbion raises $9.2 million Series A',
+  date: new Date('2019-08-30'),
+  // eslint-disable-next-line max-len
+  text: 'WASHINGTON — Orbion Space Technology announced Aug. 28 it raised a $9.2 million Series A round to develop and mass produce Hall-effect plasma thrusters for small satellites. Material Impact, a venture capital firm focused on early-stage investments, led the round, with Invest Michigan, Invest Detroit, Wakestream Ventures, Ann Arbor Spark, and Boomerang Catapult also participating. Orbion, a Houghton, Michigan-based company founded in 2016, joins a list of firms that have secured investor dollars for electric smallsat propulsion systems over the past two years, including Accion Systems in Boston, Indian startup Bellatrix Aerospace, and French startups ExoTrail and ThrustMe. Orbion’s distinction is in its approach to manufacturing, CEO Brad King said in an interview. The company is modelling its Aurora thruster production after the approach defense contractors use to build tactical missiles, he said.',
+};
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const today = new Date(date);
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
```

Comment:
Why do you need the `today` variable? 
You can use the date which you receive in props directly in `dateISO` and `dateLocalString`

`const dateISO = date.toISOString().slice(0, 10);`
`const dateLocaleString = date.toLocaleDateString`

Reply:
agree!